feature_engine/encoding/decision_tree.py

```diff
@@ -81,6 +82,11 @@ class DecisionTreeEncoder(BaseCategoricalTransformer):
         type object or categorical. If True, the encoder will select all variables or
         accept all variables entered by the user, including those cast as numeric.
 
+    regression: bool, default=False
```

Comment:
Can we replace this with the original docstring? we can find it [here](https://github.com/feature-engine/feature_engine/blob/main/feature_engine/encoding/decision_tree.py#L60).